app/assets/stylesheets/header-footer-only-print.scss

```diff
@@ -2,8 +2,6 @@
 $govuk-compatibility-govuktemplate: true;
 $govuk-use-legacy-palette: false;
 
-@import "govuk_publishing_components/components/print/feedback";
```

Comment:
Do we not need the print styles for the feedback component somewhere? Like in a new error page print stylesheet?